### DISCUSSION TITLE
Kubernetes Support

### DISCUSSION THREAD

**Discussion by trhumphries** [microsoft/ApplicationInsights-SnapshotCollector#7] [General]
Any plans for support .netcore3.1 apps that run in AKS?

**Comment by pharring** [comment]
Linux or Windows containers?

For Linux containers, the story is the same as issue #2 and I don't have anything new to announce.

For Windows containers, I believe it's technically possible today. Note that, even for .NET core apps, the container must also have the full .NET Framework installed since the SnapshotUploader process is .NET Framework. (Meaning you can't use NanoServer images).
